build/Targets/Packages.props

```diff
@@ -49,6 +49,7 @@
     <MicrosoftNETTestSdkVersion>15.3.0</MicrosoftNETTestSdkVersion>
     <MicrosoftNetCompilersVersion>2.3.1</MicrosoftNetCompilersVersion>
     <MicrosoftNetRoslynDiagnosticsVersion>2.3.0-beta2-62118-03</MicrosoftNetRoslynDiagnosticsVersion>
+    <MicrosoftNetRoslynDiagnosticsVersionForBootstrapCompiler>2.5.0-beta1-62011-01</MicrosoftNetRoslynDiagnosticsVersionForBootstrapCompiler>
```

Comment:
@jinujoseph We should bump our analyzer package version from 2.5.X to 2.6.X to match underlying Roslyn assembly versions.